stactools_sentinel/stactools/sentinel/constants.py

```diff
@@ -0,0 +1,92 @@
+import pystac
+from pystac.link import (Link, LinkType)
+from pystac.extensions.eo import Band
+
+SENTINEL_LICENSE = Link(
+    rel='license',
+    target=
+    'https://sentinel.esa.int/documents/247904/690755/Sentinel_Data_Legal_Notice',
+    link_type=LinkType.ABSOLUTE)
+
+SENTINEL_INSTRUMENTS = ['msi']
+SENTINEL_CONSTELLATION = 'Sentinel 2'
+
+SENTINEL_PROVIDER = pystac.Provider(
+    name='ESA',
+    roles=['producer', 'processor', 'licensor'],
+    url='https://earth.esa.int/web/guest/home')
+
+SENTINEL_BANDS = {
+    'B01':
+    Band.create(name='B01',
+                common_name='coastal',
+                description='Band 1 - Coastal aerosol',
+                center_wavelength=0.443,
+                full_width_half_max=0.027),
+    'B02':
+    Band.create(name='B02',
+                common_name='blue',
+                description='Band 2 - Blue',
+                center_wavelength=0.490,
+                full_width_half_max=0.098),
+    'B03':
+    Band.create(name='B03',
+                common_name='green',
+                description='Band 3 - Green',
+                center_wavelength=0.560,
+                full_width_half_max=0.045),
+    'B04':
+    Band.create(name='B04',
+                common_name='red',
+                description='Band 4 - Red',
+                center_wavelength=0.665,
+                full_width_half_max=0.038),
+    'B05':
+    Band.create(name='B05',
+                common_name='rededge1',
```

Comment:
These common_names (rededgeX, watervapor) are not part of the STAC list, and don't follow the convention of naming.
It would be nice to revisit that table and add some additional bands, but the red edge bands are more difficult because of how narrow they are and unique they tend to be for each given instrument.
In the meantime, you can leave off common_name for any band that doesn't have one:
https://github.com/radiantearth/stac-spec/tree/master/extensions/eo#common-band-names